package.json

```diff
@@ -5,6 +5,7 @@
     "babel-preset-es2015": "^6.3.13",
     "bluebird": "^3.0.5",
     "body-parser": "^1.14.1",
+    "co": "^4.6.0",
```

Comment:
That does seem odd! `Promise` in this version of Node has `.all` method and bluebird itself ships with a `.all` method :-/